### DISCUSSION TITLE
What Value do you put for Length in Quotes

### DISCUSSION THREAD

**Discussion by tajhans** [monkeytypegame/monkeytype#2635] [Questions] [answered]
What value do you put for length in quotes?

**Comment by unrealapex** [comment, marked as answer]
For the quote length, you put the length in characters of the `text`(the quote text) property:
```javascript
{
  text: "The quick brown fox jumped over the lazy dog",
  source: "Unknown",
  length: 44,
  id: 1
}
```

You can get the character count of any string in JavaScript using the `length` property(useful for long strings where you can't count manually or have your text editor show you when you highlight it):
```javascript
let x = "The quick brown fox jumped over the lazy dog";
// 44
console.log(x.length);
```

**Reply by tajhans** [reply]
Sorry for the late reply, but thanks!
